api.lua

```diff
@@ -45,6 +45,30 @@ local theme_inv = [[
 		image[5,5.2;1,1;gui_hb_bg.png]
 		image[6,5.2;1,1;gui_hb_bg.png]
 		image[7,5.2;1,1;gui_hb_bg.png]
+		image[0,6.32;1,1;gui_hb_bg.png]
```

Comment:
Use a loop.

Reply:
I can use a loop for 7y and 8y and 6y is a bit confusing.